code/modules/surgery/surgery.dm

```diff
@@ -152,7 +152,7 @@
 	if (isnull(patient_turf))
 		return null
 
-	var/obj/structure/table/optable/operating_table = locate(/obj/structure/table/optable, patient_turf)
```

Comment:
I'm sure you already know this but this is valid locate syntax it's just undocumented in the dm reference

Reply:
Yeah I checked that this code worked but its the only use of this syntax and it can be very confusing (I just ran a regex)

Reply:
Not the only
![image](https://github.com/user-attachments/assets/c969ff3e-6ecf-4d51-a898-c4f7af5031c3)

Reply:
bleugh, seems my regex was bad. I'll revert this change then and leave the PR only for heretic spells